config/routes.rb

```diff
@@ -1,4 +1,5 @@
 Rails.application.routes.draw do
+  resources :foods
```

Comment:
limit this to only the routes you need. This generates 7 restful routes for foods. I think you only need 1.

Reply:
thank you for pointing this out.  I used the  'rails g resource ___' command and forgot some of the dangers/loose ends